helm_deploy/hmpps-approved-premises-ui/values.yaml

```diff
@@ -67,6 +67,7 @@ generic-service:
     groups:
       - internal
       - probation
+    palo-alto-prisma-access-egress: "128.77.75.64/26"
```

Comment:
Sorry to be a pedant but please could we use single quotes here as that's what's used throughout the rest of the file

Reply:
i'm all for pedantry :D